### PR TITLE
refactor(web): project settings page

### PR DESCRIPTION
**Pull request by airslice**
# Overview

This PR refactors project settings page:
- Removed workspace assets page.
- Removed `PluginAccordion`.
- Refactor some on ui components `IconButton` `ModalPanel` `Tabs` `Button` `Collapse` `Icon`
- Add new ui component `Markdown`
- Add a common ui `Sidebar` and used in Dashboard sidebar.

## Summary by CodeRabbit

- **New Features**
	- Enhanced file type handling with new constants for image and file types.
	- Updated dashboard to indicate disabled tab states for clearer user feedback.
	- Introduced a new modular sidebar structure for improved navigation.
	- Added a new `PluginListItem` component for managing installed plugins.
	- Revised `ProjectSettings` to include icons in tabs for better visual representation.
	- Updated `GeneralSettings` for a more streamlined interface and user experience.
	- Launched a new `Markdown` component for dynamic and styled Markdown rendering.

- **Bug Fixes**
	- Removed deprecated project archiving functionality to streamline project management.

- **Documentation**
	- Updated user interface strings for clarity in project and plugin management.

- **Style**
	- Enhanced the styling of several components for improved visual consistency and responsiveness.

### CODE REVIEW

File: web/src/beta/lib/reearth-ui/components/Markdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  children?: string;
+  styles?: Typography;
+  backgroundColor?: string;
+  onClick?: () => void;
+  onDoubleClick?: () => void;
+};
+
+const plugins = [gfm];
+
+export const Markdown: React.FC<Props> = ({
```
Comment (mkumbobeaty) [inline web/src/beta/lib/reearth-ui/components/Markdown/index.tsx:20]:
```suggestion
export const Markdown:FC<Props> = ({
```

Comment (airslice) [reply web/src/beta/lib/reearth-ui/components/Markdown/index.tsx]: Updated.

File: web/src/beta/features/ProjectSettings/innerPages/GeneralSettings/index.tsx [hunk 2]
```diff
@@ -34,15 +40,10 @@ type Props = {
...
   onDeleteProject: () => void;
 };
-const GeneralSettings: React.FC<Props> = ({
-  project,
-  onUpdateProject,
-  onArchiveProject,
-  onDeleteProject,
-}) => {
+
+const GeneralSettings: React.FC<Props> = ({ project, onUpdateProject, onDeleteProject }) => {
```
[the remark this answers is not in the document]
Comment (airslice) [reply web/src/beta/features/ProjectSettings/innerPages/GeneralSettings/index.tsx]: Update.

File: web/src/beta/features/ProjectSettings/innerPages/GeneralSettings/index.tsx [hunk 3]
```diff
@@ -84,89 +71,51 @@ const GeneralSettings: React.FC<Props> = ({
...
-              <Text size="body" weight="bold">
+              <Typography size="body" weight="bold">
                 {t("Delete this project")}
-              </Text>
-              <Text size="body">
-                {t("Once you delete a project, there is no going back. Please be sure.")}
-              </Text>
+              </Typography>
+              <Typography size="body">
+                {t("This process will remove this project to Recycle bin.")}
```
[the remark this answers is not in the document]
Comment (airslice) [reply web/src/beta/features/ProjectSettings/innerPages/GeneralSettings/index.tsx]: Updated.

**Pull request merged**